components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <article class="bx--grid page-section qiskit-capabilities-section">
+    <h2 v-text="title" />
+  </article>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class TheQiskitCapabilitiesSection extends Vue {
+  @Prop(String) title!: string
```

Comment:
I believe we don't need to pass this information as a prop since it won't change depending on its context (as it's only found in 1 context: the homepage), so it doesn't need to be dynamic.

```suggestion
```